website/models.py

```diff
@@ -156,7 +157,7 @@ def save(self, **kwargs):
         Does not return until unique slug is generated.    
     """
     def gen_slug(self):
-        slug = str(uuid.uuid5(uuid.NAMESPACE_URL, str(self.pk)))
+        slug = str(uuid.uuid4())
```

Comment:
Just so you are aware, I ran into a bug with using self.pk for uuid5.  My quickfix was to switch to uuid4 since we would not rely on using the pk this avoids me having to do a double save which is where I think the bug was stemming from